src/commonMain/kotlin/exchange.dydx.abacus/output/Market.kt

```diff
@@ -324,6 +328,8 @@ data class MarketPerpetual(
                             nextFundingAtMilliseconds,
                             openInterest,
                             openInterestUSDC ?: 0.0,
+                            openInterestLowerCap,
```

Comment:
nit: can we update this to use named parameters?

Reply:
honestly didn't know kotlin supported that. i was actually thinking to myself "damn i wish we could use kwargs/named args"